utils/BUILD

```diff
@@ -37,3 +37,12 @@ cc_library(
         "@entangled//common/model:transaction",
     ],
 )
+
+cc_library(
+    name = "bundle_array",
+    srcs = ["bundle_array.h"],
+    deps = [
+        "@com_github_uthash//:uthash",
```

Comment:
Why did you specify `com_github_uthash`?

Reply:
Since I use `utarray` which is part of `uthash`

Reply:
Is it consistent for GitHub-based repository naming?

Reply:
We included `utarray` with the same naming in `accelerator`, so does `entangled`. `entangled` include `utarray` witch the same name.

Reply:
So, it seems to be redundant to name after "com_github_uthash". Let's clean up.